### PR TITLE
Addition to PDF-Documentation: Required Links

### PR DESCRIPTION
**Pull request by Slidr666**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

**Comment by Slidr666** [comment]
Added PanzerObserver

File: printer_mods/Slidr/PanzerBalls2.4/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Why do i need Panzerballs?
+- You probably don´t need them unless you got problems with sourcing the normal rubber compressor feet 
+or you want something more squishy with maybe a better noise reduction.
+
+How do i install Panzerballs?
+- Panzerballs are a drop in replacement for the base plate
+- You need:
+  - 4x squash balls (super slow, double yellow dot, diameter 39.5-40.5 mm)
+    o german source:https://www.decathlon.de/p/squashballe-sb-990-2er-pack-zwei-gelbe-punkte/_/R-p-312192
```
Comment (cbc02009) [inline printer_mods/Slidr/PanzerBalls2.4/readme.md:16]: these o's need to be replaced with - to format correctly

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by cbc02009** [comment]
can you open a new PR with a fresh pull of the remote to do this PR? it's so far behind that the merge conflicts are difficult to deal with.

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**